jujugui/static/gui/src/app/components/inspector/inspector.js

```diff
@@ -24,8 +24,10 @@ YUI.add('inspector-component', function() {
 
     render: function() {
       return (
-        <div>
-          Inspector
+        <div className="inspector-view">
+          <div className="inspector-content">
+            {this.props.children}
```

Comment:
WRT above comment. Instead of this rendering the children here it would look something like...
```
<components.InspectorHeader />
{_showInspectorBodyComponentFromState()}